Issue with "Datastore" addons

**Describe the bug**
All "Datastore" addons do not remain displayed.


**To Reproduce**
1. Install entire suite of "Datastore" (18 addons)
      DataStore
      DataStore_Achievements
      DataStore_Agenda
      DataStore_Auctions
      DataStore_Characters
      DataStore_Containers
      DataStore_Crafts
      DataStore_Currencies
      DataStore_Garrisons
      DataStore_Inventory
      DataStore_Keystones
      DataStore_Mails
      DataStore_Pets
      DataStore_Quests
      DataStore_Rares
      DataStore_Reputations
      DataStore_Spells
      DataStore_Stats
      DataStore_Talents

2. Check number of addons loaded > correctly updated > +18 (from 79 to 97 in my case).

3. Press "Refresh"

4. All "Datastore" are now hidden and shows "install" again in the catalog.


**Expected behavior**
This suite is a sort of dependencies for "Altoholic" addon, but are also standalone addons and are currently updated separately by the maintainer, so they should be treated as such. 


**Software involved**
 - OS: Windows 10 20H2
 - Ajour version: 0.4.3
 - Retail, Retail Beta, Retail PTR


**Additional context**
Ajour 0.4.2 display them but with "Repair" mention.
Maybe an API issue but I prefer to report it anyway.

**Log Output**

> 12:12:51.326 [ajour][INFO] Ajour 0.4.3 has started.
12:12:51.328 [ajour::gui][DEBUG] antialiasing: true
12:12:51.344 [ajour_core::config][DEBUG] loading config
12:12:51.344 [ajour_core::utility][DEBUG] checking for application update
12:12:51.344 [ajour_core::theme][DEBUG] loading user themes
12:12:51.378 [ajour_core::fs::theme][DEBUG] loaded 0 user themes
12:12:51.837 [ajour::gui::update][DEBUG] Message::Parse
12:12:51.838 [ajour::gui::update][DEBUG] config loaded:
Config {
    wow: Wow {
        directory: Some(
            "C:\\World of Warcraft",
        ),
        flavor: Retail,
    },
    addons: Addons {
        ignored: {
            RetailPTR: [],
            RetailBeta: [],
            Retail: [],
        },
        release_channels: {
            RetailBeta: {
                "XLoot": Alpha,
                "AddonUsage": Alpha,
                "Chatter": Alpha,
                "DataStore_Containers": Alpha,
                "CaerdonWardrobe": Alpha,
                "HandyNotes": Alpha,
                "!BugGrabber": Alpha,
                "Overachiever": Alpha,
                "WorldQuestTab": Alpha,
                "DataStore_Currencies": Alpha,
                "DataStore_Achievements": Alpha,
                "ParagonReputation": Alpha,
                "PitBull4": Alpha,
                "SavedInstances": Alpha,
                "SharedMedia_MyMedia": Alpha,
                "BasicMinimap": Alpha,
                "HandyNotes_ScourgeInvasionShadowlands": Alpha,
                "PreciseWorldQuestTimers": Alpha,
                "Masque_Apathy": Alpha,
                "Bartender4": Alpha,
                "TipTacTalents": Alpha,
                "DataStore_Agenda": Alpha,
                "HandyNotes_DraenorTreasures": Alpha,
                "DataStore_Characters": Alpha,
                "MasterPlan": Alpha,
                "Omen": Alpha,
                "DataStore_Garrisons": Alpha,
                "HandyNotes_Shadowlands": Alpha,
                "ActionBarProfiles": Alpha,
                "WarPlan": Alpha,
                "DataStore_Mails": Alpha,
                "InFlight": Alpha,
                "Broker_Everything": Alpha,
                "NPCScan": Alpha,
                "DBM-Challenges": Alpha,
                "Rematch": Alpha,
                "DataStore_Keystones": Alpha,
                "BugSack": Alpha,
                "AskMrRobot": Alpha,
                "MoveAnything": Alpha,
                "OmniCC": Alpha,
                "DataStore_Spells": Alpha,
                "BadBoy_History": Alpha,
                "DataStore_Talents": Alpha,
                "MegaMacro": Alpha,
                "Baggins": Alpha,
                "DBM-Core": Alpha,
                "WarCampaignsComplete": Alpha,
                "KibsItemLevelContinued": Alpha,
                "DataStore_Pets": Alpha,
                "Postal": Alpha,
                "MuteSoundFile": Alpha,
                "HideChatScroll": Alpha,
                "CookyItemSplitter": Alpha,
                "TipTacItemRef": Alpha,
                "Skada": Alpha,
                "BadBoy_Levels": Alpha,
                "DataStore_Quests": Alpha,
                "Masque": Alpha,
                "MacroToolkitIcons": Alpha,
                "DataStore_Inventory": Alpha,
                "DataStore_Reputations": Alpha,
                "TipTacOptions": Alpha,
                "TipTac": Alpha,
                "Quartz": Alpha,
                "DataStore_Rares": Alpha,
                "BadBoy": Alpha,
                "InFlight_Load": Alpha,
                "Altoholic": Alpha,
                "Raven": Alpha,
                "VuhDo": Alpha,
                "ClassOrderHallsComplete": Alpha,
                "Mapster": Alpha,
                "DataStore_Stats": Alpha,
                "GlobalIgnoreList": Alpha,
                "DataStore": Alpha,
                "LargerMacroIconSelection": Alpha,
                "DejaCharacterStats": Alpha,
                "BadBoy_CCleaner": Alpha,
                "Bazooka": Alpha,
                "CopyPasta": Alpha,
                "BeQuiet": Alpha,
                "ColorPickerPlus": Alpha,
                "HandyNotes_PandariaTreasures": Alpha,
                "WorldBossTimers": Alpha,
                "DataStore_Crafts": Alpha,
                "NeatPlates": Alpha,
                "MSBTOptions": Alpha,
                "SellJunk": Alpha,
                "DataStore_Auctions": Alpha,
                "AdvancedInterfaceOptions": Alpha,
            },
            Retail: {
                "NPCScan": Alpha,
                "DataStore_Talents": Alpha,
                "DataStore_Characters": Alpha,
                "DataStore_Reputations": Alpha,
                "MoveAnything": Alpha,
                "GlobalIgnoreList": Alpha,
                "Quartz": Alpha,
                "Baggins": Alpha,
                "BeQuiet": Alpha,
                "Rematch": Alpha,
                "MasterPlan": Alpha,
                "AddonUsage": Alpha,
                "TipTac": Alpha,
                "OmniCC": Alpha,
                "AskMrRobot": Alpha,
                "HideChatScroll": Alpha,
                "MSBTOptions": Alpha,
                "Bartender4": Alpha,
                "HandyNotes_DraenorTreasures": Alpha,
                "ActionBarProfiles": Alpha,
                "DataStore_Currencies": Alpha,
                "TipTacItemRef": Alpha,
                "ClassOrderHallsComplete": Alpha,
                "BasicMinimap": Alpha,
                "CookyItemSplitter": Alpha,
                "WorldBossTimers": Alpha,
                "HandyNotes_ScourgeInvasionShadowlands": Alpha,
                "DataStore_Pets": Alpha,
                "Bazooka": Alpha,
                "WarCampaignsComplete": Alpha,
                "BadBoy": Alpha,
                "DBM-Core": Alpha,
                "InFlight": Alpha,
                "MacroToolkitIcons": Alpha,
                "HandyNotes_Shadowlands": Alpha,
                "Raven": Alpha,
                "DataStore_Crafts": Alpha,
                "Mapster": Alpha,
                "XLoot": Alpha,
                "ParagonReputation": Alpha,
                "Masque": Alpha,
                "DataStore": Alpha,
                "KibsItemLevelContinued": Alpha,
                "CaerdonWardrobe": Alpha,
                "DataStore_Rares": Alpha,
                "MegaMacro": Alpha,
                "VuhDo": Alpha,
                "DataStore_Achievements": Alpha,
                "CopyPasta": Alpha,
                "BadBoy_History": Alpha,
                "BadBoy_CCleaner": Alpha,
                "AdvancedInterfaceOptions": Alpha,
                "InFlight_Load": Alpha,
                "WarPlan": Alpha,
                "Masque_Apathy": Alpha,
                "DataStore_Keystones": Alpha,
                "SharedMedia_MyMedia": Alpha,
                "DataStore_Inventory": Alpha,
                "Altoholic": Alpha,
                "DBM-Challenges": Alpha,
                "SavedInstances": Alpha,
                "TipTacTalents": Alpha,
                "DataStore_Stats": Alpha,
                "BadBoy_Levels": Alpha,
                "Overachiever": Alpha,
                "DataStore_Auctions": Alpha,
                "ColorPickerPlus": Alpha,
                "DataStore_Spells": Alpha,
                "DataStore_Garrisons": Alpha,
                "DataStore_Quests": Alpha,
                "NeatPlates": Alpha,
                "MuteSoundFile": Alpha,
                "SellJunk": Alpha,
                "Chatter": Alpha,
                "DataStore_Containers": Alpha,
                "LargerMacroIconSelection": Alpha,
                "Postal": Alpha,
                "HandyNotes_PandariaTreasures": Alpha,
                "PitBull4": Alpha,
                "HandyNotes": Alpha,
                "!BugGrabber": Alpha,
                "Broker_Everything": Alpha,
                "TipTacOptions": Alpha,
                "DataStore_Agenda": Alpha,
                "DataStore_Mails": Alpha,
                "WorldQuestTab": Alpha,
                "DejaCharacterStats": Alpha,
                "Skada": Alpha,
                "Omen": Alpha,
                "BugSack": Alpha,
                "PreciseWorldQuestTimers": Alpha,
            },
            RetailPTR: {
                "Mapster": Alpha,
                "SellJunk": Alpha,
                "HandyNotes": Alpha,
                "ClassOrderHallsComplete": Alpha,
                "DataStore_Reputations": Alpha,
                "DataStore_Currencies": Alpha,
                "LargerMacroIconSelection": Alpha,
                "AdvancedInterfaceOptions": Alpha,
                "DataStore_Pets": Alpha,
                "DataStore_Auctions": Alpha,
                "WarCampaignsComplete": Alpha,
                "DataStore_Mails": Alpha,
                "DataStore_Keystones": Alpha,
                "MSBTOptions": Alpha,
                "HandyNotes_Shadowlands": Alpha,
                "SavedInstances": Alpha,
                "Bazooka": Alpha,
                "Baggins": Alpha,
                "TipTacItemRef": Alpha,
                "NeatPlates": Alpha,
                "Chatter": Alpha,
                "Postal": Alpha,
                "DataStore_Agenda": Alpha,
                "BadBoy_CCleaner": Alpha,
                "Overachiever": Alpha,
                "DataStore_Inventory": Alpha,
                "MacroToolkitIcons": Alpha,
                "BadBoy": Alpha,
                "KibsItemLevelContinued": Alpha,
                "CaerdonWardrobe": Alpha,
                "MasterPlan": Alpha,
                "Raven": Alpha,
                "Altoholic": Alpha,
                "DataStore_Characters": Alpha,
                "AddonUsage": Alpha,
                "MoveAnything": Alpha,
                "InFlight": Alpha,
                "ParagonReputation": Alpha,
                "TipTacTalents": Alpha,
                "VuhDo": Alpha,
                "Broker_Everything": Alpha,
                "XLoot": Alpha,
                "Quartz": Alpha,
                "DataStore_Rares": Alpha,
                "MegaMacro": Alpha,
                "BugSack": Alpha,
                "PitBull4": Alpha,
                "TipTacOptions": Alpha,
                "ColorPickerPlus": Alpha,
                "Masque": Alpha,
                "HandyNotes_PandariaTreasures": Alpha,
                "!BugGrabber": Alpha,
                "DataStore_Garrisons": Alpha,
                "BadBoy_Levels": Alpha,
                "HideChatScroll": Alpha,
                "WorldQuestTab": Alpha,
                "SharedMedia_MyMedia": Alpha,
                "AskMrRobot": Alpha,
                "Rematch": Alpha,
                "DBM-Challenges": Alpha,
                "BeQuiet": Alpha,
                "NPCScan": Alpha,
                "DataStore_Crafts": Alpha,
                "DataStore_Containers": Alpha,
                "InFlight_Load": Alpha,
                "Skada": Alpha,
                "WarPlan": Alpha,
                "Omen": Alpha,
                "TipTac": Alpha,
                "DataStore_Stats": Alpha,
                "GlobalIgnoreList": Alpha,
                "MuteSoundFile": Alpha,
                "DejaCharacterStats": Alpha,
                "WorldBossTimers": Alpha,
                "HandyNotes_DraenorTreasures": Alpha,
                "Masque_Apathy": Alpha,
                "BadBoy_History": Alpha,
                "HandyNotes_ScourgeInvasionShadowlands": Alpha,
                "OmniCC": Alpha,
                "DataStore": Alpha,
                "CopyPasta": Alpha,
                "DataStore_Spells": Alpha,
                "DBM-Core": Alpha,
                "BasicMinimap": Alpha,
                "DataStore_Quests": Alpha,
                "DataStore_Talents": Alpha,
                "ActionBarProfiles": Alpha,
                "Bartender4": Alpha,
                "DataStore_Achievements": Alpha,
                "PreciseWorldQuestTimers": Alpha,
                "CookyItemSplitter": Alpha,
            },
        },
    },
    theme: Some(
        "Sort",
    ),
    column_config: V3 {
        my_addons_columns: [
            ColumnConfigV2 {
                key: "title",
                width: None,
                hidden: false,
            },
            ColumnConfigV2 {
                key: "status",
                width: Some(
                    124,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "local",
                width: Some(
                    176,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "remote",
                width: Some(
                    181,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "channel",
                width: Some(
                    85,
                ),
                hidden: true,
            },
            ColumnConfigV2 {
                key: "date_released",
                width: Some(
                    125,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "game_version",
                width: Some(
                    75,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "author",
                width: Some(
                    208,
                ),
                hidden: false,
            },
        ],
        catalog_columns: [
            ColumnConfigV2 {
                key: "addon",
                width: Some(
                    150,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "description",
                width: None,
                hidden: false,
            },
            ColumnConfigV2 {
                key: "source",
                width: Some(
                    85,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "num_downloads",
                width: Some(
                    105,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "date_released",
                width: Some(
                    105,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "install",
                width: Some(
                    85,
                ),
                hidden: false,
            },
        ],
    },
    window_size: Some(
        (
            1700,
            700,
        ),
    ),
    scale: Some(
        1.3000000000000003,
    ),
    backup_directory: None,
}
12:12:51.838 [ajour::gui::update][DEBUG] preparing to parse addons in "C:\\World of Warcraft\\_retail_\\Interface/AddOns"
12:12:51.838 [ajour::gui::update][DEBUG] preparing to parse addons in "C:\\World of Warcraft\\_ptr_\\Interface/AddOns"
12:12:51.838 [ajour::gui::update][DEBUG] preparing to parse addons in "C:\\World of Warcraft\\_beta_\\Interface/AddOns"
12:12:51.838 [ajour::gui::update][DEBUG] preparing to parse addons in "C:\\World of Warcraft\\_classic_\\Interface/AddOns"
12:12:51.839 [ajour::gui::update][DEBUG] preparing to parse addons in "C:\\World of Warcraft\\_classic_ptr_\\Interface/AddOns"
12:12:51.839 [ajour::gui::update][DEBUG] Message::ThemesLoaded(0 themes)
12:12:51.839 [ajour::gui::update][DEBUG] Message::NeedsUpdate(None)
12:12:51.839 [ajour_core::parse][DEBUG] Retail - parsing addons folder
12:12:51.839 [ajour_core::parse][DEBUG] Retail PTR - parsing addons folder
12:12:51.839 [ajour_core::parse][DEBUG] Retail - 155 folders in AddOns directory to parse
12:12:51.840 [ajour_core::parse][DEBUG] Retail PTR - 155 folders in AddOns directory to parse
12:12:51.840 [ajour_core::parse][DEBUG] Retail Beta - parsing addons folder
12:12:51.840 [ajour_core::parse][DEBUG] Retail Beta - 155 folders in AddOns directory to parse
12:12:51.846 [ajour_core::parse][DEBUG] Classic - parsing addons folder
12:12:51.846 [ajour_core::parse][DEBUG] Classic PTR - parsing addons folder
12:12:51.849 [ajour::gui::update][ERROR] Message::ParsedAddons(Classic) - Addon directory not found: "C:\\World of Warcraft\\_classic_\\Interface/AddOns"
12:12:51.849 [ajour::gui::update][ERROR] Message::ParsedAddons(Classic PTR) - Addon directory not found: "C:\\World of Warcraft\\_classic_ptr_\\Interface/AddOns"
12:12:51.963 [ajour_core::parse][DEBUG] Retail - 155 fingerprints: 155 cached, 0 calculated, 0 added, 0 removed
12:12:52.018 [ajour_core::parse][DEBUG] Retail - 154 successfully parsed from '.toc'
12:12:52.018 [ajour_core::parse][DEBUG] Retail - 0 addons with tukui id
12:12:52.018 [ajour_core::parse][DEBUG] Retail - 0 addons from tukui package metadata
12:12:52.018 [ajour_core::parse][DEBUG] Retail - 154 unique fingerprints to check against curse api
12:12:52.021 [ajour_core::parse][DEBUG] Retail PTR - 155 fingerprints: 155 cached, 0 calculated, 0 added, 0 removed
12:12:52.070 [ajour_core::parse][DEBUG] Retail PTR - 154 successfully parsed from '.toc'
12:12:52.070 [ajour_core::parse][DEBUG] Retail PTR - 0 addons with tukui id
12:12:52.070 [ajour_core::parse][DEBUG] Retail PTR - 0 addons from tukui package metadata
12:12:52.070 [ajour_core::parse][DEBUG] Retail PTR - 154 unique fingerprints to check against curse api
12:12:52.075 [ajour_core::parse][DEBUG] Retail Beta - 155 fingerprints: 155 cached, 0 calculated, 0 added, 0 removed
12:12:52.136 [ajour_core::parse][DEBUG] Retail Beta - 154 successfully parsed from '.toc'
12:12:52.136 [ajour_core::parse][DEBUG] Retail Beta - 0 addons with tukui id
12:12:52.136 [ajour_core::parse][DEBUG] Retail Beta - 0 addons from tukui package metadata
12:12:52.136 [ajour_core::parse][DEBUG] Retail Beta - 154 unique fingerprints to check against curse api
12:12:52.412 [ajour::gui::update][DEBUG] Message::CatalogDownloaded(8668 addons in catalog)
12:12:56.319 [ajour_core::parse][DEBUG] Retail PTR - 88 exact fingerprint matches against curse api
12:12:56.319 [ajour_core::parse][DEBUG] Retail PTR - 88 addons from fingerprint metadata
12:12:56.319 [ajour_core::parse][DEBUG] Retail PTR - 89 addons with curse id
12:12:56.860 [ajour_core::parse][DEBUG] Retail PTR - 88 addons updated with curse id package metadata
12:12:56.860 [ajour_core::parse][DEBUG] Retail PTR - 1 addons created from curse id package metadata
12:12:56.860 [ajour_core::parse][DEBUG] Retail PTR - 89 addons successfully parsed
12:12:56.860 [ajour_core::parse][DEBUG] Retail PTR - 9 unknown addons
12:12:56.864 [ajour::gui::update][DEBUG] Message::ParsedAddons(Retail PTR, 79 addons)
12:12:57.204 [ajour_core::parse][DEBUG] Retail - 88 exact fingerprint matches against curse api
12:12:57.206 [ajour_core::parse][DEBUG] Retail - 88 addons from fingerprint metadata
12:12:57.206 [ajour_core::parse][DEBUG] Retail - 89 addons with curse id
12:12:57.676 [ajour_core::parse][DEBUG] Retail - 88 addons updated with curse id package metadata
12:12:57.676 [ajour_core::parse][DEBUG] Retail - 1 addons created from curse id package metadata
12:12:57.676 [ajour_core::parse][DEBUG] Retail - 89 addons successfully parsed
12:12:57.676 [ajour_core::parse][DEBUG] Retail - 9 unknown addons
12:12:57.678 [ajour::gui::update][DEBUG] Message::ParsedAddons(Retail, 79 addons)
12:12:58.043 [ajour_core::parse][DEBUG] Retail Beta - 88 exact fingerprint matches against curse api
12:12:58.044 [ajour_core::parse][DEBUG] Retail Beta - 88 addons from fingerprint metadata
12:12:58.044 [ajour_core::parse][DEBUG] Retail Beta - 89 addons with curse id
12:12:58.930 [ajour_core::parse][DEBUG] Retail Beta - 88 addons updated with curse id package metadata
12:12:58.930 [ajour_core::parse][DEBUG] Retail Beta - 1 addons created from curse id package metadata
12:12:58.930 [ajour_core::parse][DEBUG] Retail Beta - 89 addons successfully parsed
12:12:58.930 [ajour_core::parse][DEBUG] Retail Beta - 9 unknown addons
12:12:58.940 [ajour::gui::update][DEBUG] Message::ParsedAddons(Retail Beta, 79 addons)
12:13:25.704 [ajour::gui::update][DEBUG] Interaction::ModeSelected(Catalog)
12:13:56.214 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 19874)
12:13:56.765 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 19872)
12:13:56.965 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 19874)
12:13:56.966 [ajour_core::network][DEBUG] downloading remote version 9.0.1-009 for 19874
12:13:57.269 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 20208)
12:13:57.375 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 19872)
12:13:57.375 [ajour_core::network][DEBUG] downloading remote version 9.0.1-008 for 19872
12:13:57.495 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 19874, error: false))
12:13:57.512 [ajour::gui::update][DEBUG] Message::UnpackedAddon((19874, error: false))
12:13:57.513 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Crafts
12:13:57.529 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Crafts, error: false))
12:13:57.583 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 19872, error: false))
12:13:57.588 [ajour::gui::update][DEBUG] Message::UnpackedAddon((19872, error: false))
12:13:57.589 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Containers
12:13:57.596 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Containers, error: false))
12:13:57.759 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 20206)
12:13:58.192 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 20208)
12:13:58.192 [ajour_core::network][DEBUG] downloading remote version 9.0.1-001 for 20208
12:13:58.278 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 19873)
12:13:58.383 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 20208, error: false))
12:13:58.417 [ajour::gui::update][DEBUG] Message::UnpackedAddon((20208, error: false))
12:13:58.418 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Inventory
12:13:58.442 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Inventory, error: false))
12:13:58.520 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 20206)
12:13:58.521 [ajour_core::network][DEBUG] downloading remote version 9.0.1-002 for 20206
12:13:58.662 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 20206, error: false))
12:13:58.677 [ajour::gui::update][DEBUG] Message::UnpackedAddon((20206, error: false))
12:13:58.678 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Characters
12:13:58.687 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Characters, error: false))
12:13:58.805 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 20205)
12:13:58.911 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 19873)
12:13:58.911 [ajour_core::network][DEBUG] downloading remote version 9.0.1-001 for 19873
12:13:59.039 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 19873, error: false))
12:13:59.048 [ajour::gui::update][DEBUG] Message::UnpackedAddon((19873, error: false))
12:13:59.049 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Achievements
12:13:59.057 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Achievements, error: false))
12:13:59.305 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 20210)
12:13:59.657 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 20205)
12:13:59.657 [ajour_core::network][DEBUG] downloading remote version 9.0.1-001 for 20205
12:13:59.782 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 20205, error: false))
12:13:59.789 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 20209)
12:13:59.802 [ajour::gui::update][DEBUG] Message::UnpackedAddon((20205, error: false))
12:13:59.804 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Auctions
12:13:59.813 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Auctions, error: false))
12:14:00.155 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 20210)
12:14:00.156 [ajour_core::network][DEBUG] downloading remote version 9.0.1-001 for 20210
12:14:00.250 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 19876)
12:14:00.282 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 20210, error: false))
12:14:00.296 [ajour::gui::update][DEBUG] Message::UnpackedAddon((20210, error: false))
12:14:00.298 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Quests
12:14:00.307 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Quests, error: false))
12:14:00.598 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 20209)
12:14:00.599 [ajour_core::network][DEBUG] downloading remote version 9.0.1-001 for 20209
12:14:00.723 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 20209, error: false))
12:14:00.739 [ajour::gui::update][DEBUG] Message::UnpackedAddon((20209, error: false))
12:14:00.740 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Mails
12:14:00.744 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 19877)
12:14:00.751 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Mails, error: false))
12:14:00.965 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 19876)
12:14:00.966 [ajour_core::network][DEBUG] downloading remote version 9.0.1-003 for 19876
12:14:01.094 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 19876, error: false))
12:14:01.105 [ajour::gui::update][DEBUG] Message::UnpackedAddon((19876, error: false))
12:14:01.106 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Reputations
12:14:01.113 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Reputations, error: false))
12:14:01.263 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 19875)
12:14:01.609 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 19877)
12:14:01.610 [ajour_core::network][DEBUG] downloading remote version 9.0.1-001 for 19877
12:14:01.740 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 19877, error: false))
12:14:01.752 [ajour::gui::update][DEBUG] Message::UnpackedAddon((19877, error: false))
12:14:01.753 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Talents
12:14:01.761 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Talents, error: false))
12:14:01.839 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 20207)
12:14:01.980 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 19875)
12:14:01.981 [ajour_core::network][DEBUG] downloading remote version 9.0.1-001 for 19875
12:14:02.109 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 19875, error: false))
12:14:02.118 [ajour::gui::update][DEBUG] Message::UnpackedAddon((19875, error: false))
12:14:02.119 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Pets
12:14:02.126 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Pets, error: false))
12:14:02.350 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 20212)
12:14:02.675 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 20207)
12:14:02.676 [ajour_core::network][DEBUG] downloading remote version 9.0.1-001 for 20207
12:14:02.796 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 20207, error: false))
12:14:02.802 [ajour::gui::update][DEBUG] Message::UnpackedAddon((20207, error: false))
12:14:02.804 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Currencies
12:14:02.809 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Currencies, error: false))
12:14:02.836 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 20213)
12:14:03.236 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 20212)
12:14:03.236 [ajour_core::network][DEBUG] downloading remote version 9.0.1-001 for 20212
12:14:03.359 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 20212, error: false))
12:14:03.365 [ajour::gui::update][DEBUG] Message::UnpackedAddon((20212, error: false))
12:14:03.366 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Spells
12:14:03.371 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Spells, error: false))
12:14:03.390 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 29739)
12:14:04.149 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 29739)
12:14:04.150 [ajour_core::network][DEBUG] downloading remote version 9.0.1-011 for 29739
12:14:04.277 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 29739, error: false))
12:14:04.286 [ajour::gui::update][DEBUG] Message::UnpackedAddon((29739, error: false))
12:14:04.287 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Agenda
12:14:04.295 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Agenda, error: false))
12:14:04.525 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 20213)
12:14:04.526 [ajour_core::network][DEBUG] downloading remote version 9.0.1-001 for 20213
12:14:04.645 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 20213, error: false))
12:14:04.650 [ajour::gui::update][DEBUG] Message::UnpackedAddon((20213, error: false))
12:14:04.651 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Stats
12:14:04.657 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Stats, error: false))
12:14:04.695 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 88051)
12:14:05.064 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 411498)
12:14:05.524 [ajour::gui::update][DEBUG] Interaction::CatalogInstall(Curse, Retail, 411494)
12:14:05.687 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 88051)
12:14:05.688 [ajour_core::network][DEBUG] downloading remote version 9.0.1-001 for 88051
12:14:05.818 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 88051, error: false))
12:14:05.836 [ajour::gui::update][DEBUG] Message::UnpackedAddon((88051, error: false))
12:14:05.837 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Garrisons
12:14:05.853 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Garrisons, error: false))
12:14:06.121 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 411498)
12:14:06.122 [ajour_core::network][DEBUG] downloading remote version 9.0.1-001 for 411498
12:14:06.242 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 411498, error: false))
12:14:06.246 [ajour::gui::update][DEBUG] Message::UnpackedAddon((411498, error: false))
12:14:06.247 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Keystones
12:14:06.254 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Keystones, error: false))
12:14:06.322 [ajour::gui::update][DEBUG] Message::CatalogInstallAddonFetched(Retail, 411494)
12:14:06.323 [ajour_core::network][DEBUG] downloading remote version 9.0.1-001 for 411494
12:14:06.456 [ajour::gui::update][DEBUG] Message::DownloadedAddon((Retail, 411494, error: false))
12:14:06.473 [ajour::gui::update][DEBUG] Message::UnpackedAddon((411494, error: false))
12:14:06.474 [ajour_core::parse][DEBUG] Retail - updating fingerprint for DataStore_Rares
12:14:06.490 [ajour::gui::update][DEBUG] Message::UpdateFingerprint((Retail, DataStore_Rares, error: false))
12:14:10.618 [ajour::gui::update][DEBUG] Interaction::FlavorSelected(Retail)
12:14:12.456 [ajour::gui::update][DEBUG] Interaction::ModeSelected(MyAddons)
12:14:39.638 [ajour::gui::update][DEBUG] Interaction::Refresh
12:14:39.639 [ajour_core::config][DEBUG] loading config
12:14:39.641 [ajour::gui::update][DEBUG] Message::Parse
12:14:39.641 [ajour::gui::update][DEBUG] config loaded:
Config {
    wow: Wow {
        directory: Some(
            "C:\\World of Warcraft",
        ),
        flavor: Retail,
    },
    addons: Addons {
        ignored: {
            RetailPTR: [],
            Retail: [],
            RetailBeta: [],
        },
        release_channels: {
            RetailBeta: {
                "MegaMacro": Alpha,
                "WarCampaignsComplete": Alpha,
                "XLoot": Alpha,
                "PitBull4": Alpha,
                "BadBoy_Levels": Alpha,
                "DataStore_Quests": Alpha,
                "MasterPlan": Alpha,
                "MacroToolkitIcons": Alpha,
                "Mapster": Alpha,
                "SharedMedia_MyMedia": Alpha,
                "DataStore_Currencies": Alpha,
                "Rematch": Alpha,
                "DataStore_Pets": Alpha,
                "DataStore_Stats": Alpha,
                "Omen": Alpha,
                "VuhDo": Alpha,
                "GlobalIgnoreList": Alpha,
                "DataStore_Crafts": Alpha,
                "DataStore_Agenda": Alpha,
                "DataStore_Garrisons": Alpha,
                "TipTac": Alpha,
                "MSBTOptions": Alpha,
                "HideChatScroll": Alpha,
                "BadBoy_CCleaner": Alpha,
                "!BugGrabber": Alpha,
                "HandyNotes_Shadowlands": Alpha,
                "DataStore_Characters": Alpha,
                "DataStore_Containers": Alpha,
                "OmniCC": Alpha,
                "DataStore_Talents": Alpha,
                "DataStore_Inventory": Alpha,
                "DataStore_Keystones": Alpha,
                "WorldBossTimers": Alpha,
                "AskMrRobot": Alpha,
                "SavedInstances": Alpha,
                "HandyNotes_DraenorTreasures": Alpha,
                "CookyItemSplitter": Alpha,
                "InFlight_Load": Alpha,
                "DataStore_Auctions": Alpha,
                "ParagonReputation": Alpha,
                "BadBoy_History": Alpha,
                "DataStore_Reputations": Alpha,
                "HandyNotes_ScourgeInvasionShadowlands": Alpha,
                "TipTacItemRef": Alpha,
                "BasicMinimap": Alpha,
                "DataStore": Alpha,
                "ActionBarProfiles": Alpha,
                "SellJunk": Alpha,
                "DBM-Challenges": Alpha,
                "Skada": Alpha,
                "Baggins": Alpha,
                "CaerdonWardrobe": Alpha,
                "Raven": Alpha,
                "HandyNotes": Alpha,
                "Masque_Apathy": Alpha,
                "CopyPasta": Alpha,
                "Postal": Alpha,
                "AddonUsage": Alpha,
                "LargerMacroIconSelection": Alpha,
                "KibsItemLevelContinued": Alpha,
                "HandyNotes_PandariaTreasures": Alpha,
                "AdvancedInterfaceOptions": Alpha,
                "Quartz": Alpha,
                "DataStore_Spells": Alpha,
                "TipTacOptions": Alpha,
                "BeQuiet": Alpha,
                "MuteSoundFile": Alpha,
                "Broker_Everything": Alpha,
                "DBM-Core": Alpha,
                "TipTacTalents": Alpha,
                "WarPlan": Alpha,
                "DataStore_Achievements": Alpha,
                "NPCScan": Alpha,
                "DataStore_Rares": Alpha,
                "DataStore_Mails": Alpha,
                "NeatPlates": Alpha,
                "Masque": Alpha,
                "BadBoy": Alpha,
                "ClassOrderHallsComplete": Alpha,
                "BugSack": Alpha,
                "Bazooka": Alpha,
                "Bartender4": Alpha,
                "DejaCharacterStats": Alpha,
                "Altoholic": Alpha,
                "MoveAnything": Alpha,
                "PreciseWorldQuestTimers": Alpha,
                "WorldQuestTab": Alpha,
                "Overachiever": Alpha,
                "ColorPickerPlus": Alpha,
                "Chatter": Alpha,
                "InFlight": Alpha,
            },
            RetailPTR: {
                "DataStore_Pets": Alpha,
                "WorldBossTimers": Alpha,
                "BadBoy_CCleaner": Alpha,
                "Raven": Alpha,
                "VuhDo": Alpha,
                "AddonUsage": Alpha,
                "AdvancedInterfaceOptions": Alpha,
                "TipTacOptions": Alpha,
                "DataStore_Crafts": Alpha,
                "MSBTOptions": Alpha,
                "!BugGrabber": Alpha,
                "InFlight_Load": Alpha,
                "Mapster": Alpha,
                "TipTacItemRef": Alpha,
                "DBM-Challenges": Alpha,
                "Broker_Everything": Alpha,
                "ColorPickerPlus": Alpha,
                "Masque_Apathy": Alpha,
                "SellJunk": Alpha,
                "DataStore": Alpha,
                "DataStore_Spells": Alpha,
                "PreciseWorldQuestTimers": Alpha,
                "PitBull4": Alpha,
                "InFlight": Alpha,
                "HandyNotes_ScourgeInvasionShadowlands": Alpha,
                "ActionBarProfiles": Alpha,
                "BasicMinimap": Alpha,
                "WarPlan": Alpha,
                "Omen": Alpha,
                "DataStore_Mails": Alpha,
                "Baggins": Alpha,
                "DataStore_Quests": Alpha,
                "DataStore_Auctions": Alpha,
                "ClassOrderHallsComplete": Alpha,
                "Overachiever": Alpha,
                "ParagonReputation": Alpha,
                "WorldQuestTab": Alpha,
                "KibsItemLevelContinued": Alpha,
                "BugSack": Alpha,
                "DataStore_Stats": Alpha,
                "BadBoy_History": Alpha,
                "CookyItemSplitter": Alpha,
                "Chatter": Alpha,
                "SavedInstances": Alpha,
                "WarCampaignsComplete": Alpha,
                "DataStore_Containers": Alpha,
                "Rematch": Alpha,
                "DejaCharacterStats": Alpha,
                "BeQuiet": Alpha,
                "DataStore_Reputations": Alpha,
                "DBM-Core": Alpha,
                "TipTacTalents": Alpha,
                "Bazooka": Alpha,
                "Altoholic": Alpha,
                "HandyNotes_Shadowlands": Alpha,
                "Postal": Alpha,
                "DataStore_Rares": Alpha,
                "DataStore_Talents": Alpha,
                "MacroToolkitIcons": Alpha,
                "Skada": Alpha,
                "OmniCC": Alpha,
                "DataStore_Currencies": Alpha,
                "AskMrRobot": Alpha,
                "DataStore_Agenda": Alpha,
                "BadBoy": Alpha,
                "Quartz": Alpha,
                "HandyNotes_PandariaTreasures": Alpha,
                "BadBoy_Levels": Alpha,
                "MuteSoundFile": Alpha,
                "DataStore_Keystones": Alpha,
                "Masque": Alpha,
                "Bartender4": Alpha,
                "DataStore_Achievements": Alpha,
                "HandyNotes": Alpha,
                "DataStore_Characters": Alpha,
                "DataStore_Inventory": Alpha,
                "HandyNotes_DraenorTreasures": Alpha,
                "NeatPlates": Alpha,
                "HideChatScroll": Alpha,
                "SharedMedia_MyMedia": Alpha,
                "GlobalIgnoreList": Alpha,
                "LargerMacroIconSelection": Alpha,
                "NPCScan": Alpha,
                "MasterPlan": Alpha,
                "XLoot": Alpha,
                "CaerdonWardrobe": Alpha,
                "TipTac": Alpha,
                "CopyPasta": Alpha,
                "MoveAnything": Alpha,
                "MegaMacro": Alpha,
                "DataStore_Garrisons": Alpha,
            },
            Retail: {
                "HandyNotes_ScourgeInvasionShadowlands": Alpha,
                "MuteSoundFile": Alpha,
                "MoveAnything": Alpha,
                "Bartender4": Alpha,
                "DejaCharacterStats": Alpha,
                "Raven": Alpha,
                "BugSack": Alpha,
                "TipTacOptions": Alpha,
                "CaerdonWardrobe": Alpha,
                "ClassOrderHallsComplete": Alpha,
                "InFlight": Alpha,
                "DataStore_Inventory": Alpha,
                "DBM-Challenges": Alpha,
                "TipTacTalents": Alpha,
                "DataStore_Rares": Alpha,
                "DataStore_Garrisons": Alpha,
                "AskMrRobot": Alpha,
                "OmniCC": Alpha,
                "Rematch": Alpha,
                "WorldBossTimers": Alpha,
                "DataStore_Quests": Alpha,
                "NeatPlates": Alpha,
                "SellJunk": Alpha,
                "LargerMacroIconSelection": Alpha,
                "DBM-Core": Alpha,
                "NPCScan": Alpha,
                "ParagonReputation": Alpha,
                "MegaMacro": Alpha,
                "MasterPlan": Alpha,
                "DataStore_Currencies": Alpha,
                "TipTacItemRef": Alpha,
                "HandyNotes_Shadowlands": Alpha,
                "DataStore_Talents": Alpha,
                "TipTac": Alpha,
                "DataStore_Keystones": Alpha,
                "Chatter": Alpha,
                "HandyNotes_PandariaTreasures": Alpha,
                "WarCampaignsComplete": Alpha,
                "Quartz": Alpha,
                "Mapster": Alpha,
                "BadBoy_History": Alpha,
                "Postal": Alpha,
                "DataStore_Pets": Alpha,
                "WarPlan": Alpha,
                "DataStore_Spells": Alpha,
                "XLoot": Alpha,
                "Broker_Everything": Alpha,
                "InFlight_Load": Alpha,
                "ColorPickerPlus": Alpha,
                "Skada": Alpha,
                "BadBoy_CCleaner": Alpha,
                "DataStore_Crafts": Alpha,
                "AdvancedInterfaceOptions": Alpha,
                "DataStore_Characters": Alpha,
                "AddonUsage": Alpha,
                "ActionBarProfiles": Alpha,
                "MacroToolkitIcons": Alpha,
                "CookyItemSplitter": Alpha,
                "DataStore_Auctions": Alpha,
                "DataStore_Agenda": Alpha,
                "HandyNotes": Alpha,
                "BeQuiet": Alpha,
                "DataStore_Reputations": Alpha,
                "HandyNotes_DraenorTreasures": Alpha,
                "BasicMinimap": Alpha,
                "WorldQuestTab": Alpha,
                "HideChatScroll": Alpha,
                "Masque": Alpha,
                "PitBull4": Alpha,
                "DataStore_Achievements": Alpha,
                "Omen": Alpha,
                "PreciseWorldQuestTimers": Alpha,
                "Baggins": Alpha,
                "BadBoy": Alpha,
                "Altoholic": Alpha,
                "Bazooka": Alpha,
                "DataStore": Alpha,
                "DataStore_Stats": Alpha,
                "SavedInstances": Alpha,
                "Masque_Apathy": Alpha,
                "BadBoy_Levels": Alpha,
                "GlobalIgnoreList": Alpha,
                "Overachiever": Alpha,
                "KibsItemLevelContinued": Alpha,
                "CopyPasta": Alpha,
                "DataStore_Mails": Alpha,
                "MSBTOptions": Alpha,
                "VuhDo": Alpha,
                "!BugGrabber": Alpha,
                "SharedMedia_MyMedia": Alpha,
                "DataStore_Containers": Alpha,
            },
        },
    },
    theme: Some(
        "Sort",
    ),
    column_config: V3 {
        my_addons_columns: [
            ColumnConfigV2 {
                key: "title",
                width: None,
                hidden: false,
            },
            ColumnConfigV2 {
                key: "status",
                width: Some(
                    124,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "local",
                width: Some(
                    176,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "remote",
                width: Some(
                    181,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "channel",
                width: Some(
                    85,
                ),
                hidden: true,
            },
            ColumnConfigV2 {
                key: "date_released",
                width: Some(
                    125,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "game_version",
                width: Some(
                    75,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "author",
                width: Some(
                    208,
                ),
                hidden: false,
            },
        ],
        catalog_columns: [
            ColumnConfigV2 {
                key: "addon",
                width: Some(
                    150,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "description",
                width: None,
                hidden: false,
            },
            ColumnConfigV2 {
                key: "source",
                width: Some(
                    85,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "num_downloads",
                width: Some(
                    105,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "date_released",
                width: Some(
                    105,
                ),
                hidden: false,
            },
            ColumnConfigV2 {
                key: "install",
                width: Some(
                    85,
                ),
                hidden: false,
            },
        ],
    },
    window_size: Some(
        (
            1700,
            700,
        ),
    ),
    scale: Some(
        1.3000000000000003,
    ),
    backup_directory: None,
}
12:14:39.642 [ajour::gui::update][DEBUG] preparing to parse addons in "C:\\World of Warcraft\\_retail_\\Interface/AddOns"
12:14:39.642 [ajour::gui::update][DEBUG] preparing to parse addons in "C:\\World of Warcraft\\_ptr_\\Interface/AddOns"
12:14:39.642 [ajour::gui::update][DEBUG] preparing to parse addons in "C:\\World of Warcraft\\_beta_\\Interface/AddOns"
12:14:39.642 [ajour::gui::update][DEBUG] preparing to parse addons in "C:\\World of Warcraft\\_classic_\\Interface/AddOns"
12:14:39.642 [ajour::gui::update][DEBUG] preparing to parse addons in "C:\\World of Warcraft\\_classic_ptr_\\Interface/AddOns"
12:14:39.642 [ajour_core::parse][DEBUG] Retail - parsing addons folder
12:14:39.642 [ajour_core::parse][DEBUG] Retail Beta - parsing addons folder
12:14:39.643 [ajour_core::parse][DEBUG] Retail Beta - 155 folders in AddOns directory to parse
12:14:39.643 [ajour_core::parse][DEBUG] Retail - 155 folders in AddOns directory to parse
12:14:39.643 [ajour_core::parse][DEBUG] Classic - parsing addons folder
12:14:39.643 [ajour_core::parse][DEBUG] Classic PTR - parsing addons folder
12:14:39.642 [ajour_core::parse][DEBUG] Retail PTR - parsing addons folder
12:14:39.644 [ajour_core::parse][DEBUG] Retail PTR - 155 folders in AddOns directory to parse
12:14:39.646 [ajour_core::parse][DEBUG] Retail Beta - 155 fingerprints: 155 cached, 0 calculated, 0 added, 0 removed
12:14:39.649 [ajour::gui::update][ERROR] Message::ParsedAddons(Classic) - Addon directory not found: "C:\\World of Warcraft\\_classic_\\Interface/AddOns"
12:14:39.649 [ajour::gui::update][ERROR] Message::ParsedAddons(Classic PTR) - Addon directory not found: "C:\\World of Warcraft\\_classic_ptr_\\Interface/AddOns"
12:14:39.670 [ajour_core::parse][DEBUG] Retail Beta - 154 successfully parsed from '.toc'
12:14:39.670 [ajour_core::parse][DEBUG] Retail Beta - 0 addons with tukui id
12:14:39.670 [ajour_core::parse][DEBUG] Retail Beta - 0 addons from tukui package metadata
12:14:39.670 [ajour_core::parse][DEBUG] Retail Beta - 154 unique fingerprints to check against curse api
12:14:39.672 [ajour_core::parse][DEBUG] Retail PTR - 155 fingerprints: 155 cached, 0 calculated, 0 added, 0 removed
12:14:39.694 [ajour_core::parse][DEBUG] Retail PTR - 154 successfully parsed from '.toc'
12:14:39.694 [ajour_core::parse][DEBUG] Retail PTR - 0 addons with tukui id
12:14:39.694 [ajour_core::parse][DEBUG] Retail PTR - 0 addons from tukui package metadata
12:14:39.694 [ajour_core::parse][DEBUG] Retail PTR - 154 unique fingerprints to check against curse api
12:14:39.698 [ajour_core::parse][DEBUG] Retail - 155 fingerprints: 155 cached, 0 calculated, 0 added, 0 removed
12:14:39.723 [ajour_core::parse][DEBUG] Retail - 154 successfully parsed from '.toc'
12:14:39.723 [ajour_core::parse][DEBUG] Retail - 0 addons with tukui id
12:14:39.723 [ajour_core::parse][DEBUG] Retail - 0 addons from tukui package metadata
12:14:39.723 [ajour_core::parse][DEBUG] Retail - 154 unique fingerprints to check against curse api
12:14:44.906 [ajour_core::parse][DEBUG] Retail - 88 exact fingerprint matches against curse api
12:14:44.907 [ajour_core::parse][DEBUG] Retail - 88 addons from fingerprint metadata
12:14:44.907 [ajour_core::parse][DEBUG] Retail - 89 addons with curse id
12:14:45.220 [ajour_core::parse][DEBUG] Retail Beta - 88 exact fingerprint matches against curse api
12:14:45.220 [ajour_core::parse][DEBUG] Retail Beta - 88 addons from fingerprint metadata
12:14:45.220 [ajour_core::parse][DEBUG] Retail Beta - 89 addons with curse id
12:14:45.312 [ajour_core::parse][DEBUG] Retail PTR - 88 exact fingerprint matches against curse api
12:14:45.313 [ajour_core::parse][DEBUG] Retail PTR - 88 addons from fingerprint metadata
12:14:45.313 [ajour_core::parse][DEBUG] Retail PTR - 89 addons with curse id
12:14:45.648 [ajour_core::parse][DEBUG] Retail - 88 addons updated with curse id package metadata
12:14:45.648 [ajour_core::parse][DEBUG] Retail - 1 addons created from curse id package metadata
12:14:45.648 [ajour_core::parse][DEBUG] Retail - 89 addons successfully parsed
12:14:45.649 [ajour_core::parse][DEBUG] Retail - 9 unknown addons
12:14:45.653 [ajour::gui::update][DEBUG] Message::ParsedAddons(Retail, 79 addons)
12:14:45.796 [ajour_core::parse][DEBUG] Retail Beta - 88 addons updated with curse id package metadata
12:14:45.796 [ajour_core::parse][DEBUG] Retail Beta - 1 addons created from curse id package metadata
12:14:45.796 [ajour_core::parse][DEBUG] Retail Beta - 89 addons successfully parsed
12:14:45.796 [ajour_core::parse][DEBUG] Retail Beta - 9 unknown addons
12:14:45.800 [ajour::gui::update][DEBUG] Message::ParsedAddons(Retail Beta, 79 addons)
12:14:45.859 [ajour_core::parse][DEBUG] Retail PTR - 88 addons updated with curse id package metadata
12:14:45.859 [ajour_core::parse][DEBUG] Retail PTR - 1 addons created from curse id package metadata
12:14:45.859 [ajour_core::parse][DEBUG] Retail PTR - 89 addons successfully parsed
12:14:45.859 [ajour_core::parse][DEBUG] Retail PTR - 9 unknown addons
12:14:45.865 [ajour::gui::update][DEBUG] Message::ParsedAddons(Retail PTR, 79 addons)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` and `Removed`.
 
 ## [Unreleased]
+### Fixed
+- Now correctly shows all sub-addons if they are a seperate addons.
+  - An example is Altoholic-Retail (Teelo's Fork). All it's dependencies are actually standalone addons. They are now correctly shown.
+
 ## [0.4.3] - 2020-10-22
 
 ### Fixed

--- a/crates/core/src/parse.rs
+++ b/crates/core/src/parse.rs
@@ -517,21 +517,6 @@ pub async fn read_addon_directory<P: AsRef<Path>>(
 
     concatenated.extend(unknown_addons);
 
-    // We do a extra clean up here to ensure that we dont display any addons which is a module.
-    // Idea is that we loop the addon.folders and mark all "dependencies".
-    // We then retain them from the concatenated Vec. If there is any.
-    let mut marked = vec![];
-    for addon in &concatenated {
-        for folder in &addon.folders {
-            if folder.id != addon.primary_folder_id {
-                marked.push(folder.id.clone());
-            }
-        }
-    }
-
-    // Remove dependency addons.
-    concatenated.retain(|addon| !marked.iter().any(|id| &addon.primary_folder_id == id));
-
     Ok(concatenated)
 }
 

